### ISSUE TITLE
Move away from Dynamo for social content

### ISSUE THREAD

**Issue by kyl191** [Calligre/server#70]
When I planned it, DynamoDB was a quick win:
- We don't need to worry about usage, large conferences can grow practically unbounded
- There's a bunch of usage in the free tier
- We can (fairly) arbitrarily extend posts to include other stuff
- The main reason: IT WORKED WITH LAMBDA, avoiding the weird VPC joining stuff.

In practice, it's not that great:
- The 'unbounded' growth is a lie, after 10GB of data we'll be split across 2+ partitions - and due to our desire to sort posts by time, we have a _really_ hacky solution that sorts posts within a partition.
  - The performance part doesn't _really_ affect us, expected usage is steady state ~25 qps. Dynamo only gets concerned around 1000 qps. 
- Coding is significantly harder, since we need joins with stuff in the Postgres DB
- We're hard coding what to return, so adding an extra field doesn't get us much (and introduces errors if we depend on that field existing in old posts)
- Lambda is dead, due to not playing well with psycopg2. It was easy to move all the code into the main server, but DynamoDB still worked, so it wasn't touched.
  - Neither of the Lambda functions currently talk to Dynamo either, so there's nothing holding us back there.

In short:
DynamoDB works, but our code is far from as clean as it could be if we only used Postgres. From a supportability point of view, it makes sense to consolidate to a single data storage mechanism.
